File: 944. Delete Columns to Make Sorted/944.go
package leetcode

func minDeletionSize(A []string) int {
	count := 0
	for i:=0; i<len(A[0]); i++{
		for j:=1; j<len(A); j++{
			if A[j-1][i] > A[j][i]{
				count ++
				break
			}
		}
	}
	return count
}

File: 944. Delete Columns to Make Sorted/944.c
int minDeletionSize(char ** A, int ASize){
    int len = strlen(*A);
    int count = 0;
    for (int i=0; i<len; i++){
        for (int j=1; j<ASize; j++){
            if(A[j-1][i] > A[j][i]){
                count++;
                break;
            }
        }
    }
    return count;
}